kvstore/benchmark.py

```diff
@@ -0,0 +1,262 @@
+    print("Remote pull Throughput (MB): %f" % (total_bytes / (end-start) / 1024.0 / 1024.0))
+    
+    
+    ################################# local push ######################################
+
+    
+    if args.machine_id == 1:
+        id_tensor = np.random.randint(args.graph_size, size=args.data_size)
+        id_tensor = id_tensor + args.graph_size
+    else:
+        id_tensor = np.random.randint(args.graph_size, size=args.data_size)
+    id_tensor = F.tensor(id_tensor)
+    data_tensor = F.zeros((args.data_size, args.dim), F.float32, F.cpu())
+
+    kvclient.barrier()
+    start = time.time()
+    for _ in range(100):
+        res = kvclient.push(name='data', id_tensor=id_tensor, data_tensor=data_tensor)
+    kvclient.barrier()
+    end = time.time()
+    total_bytes = (args.data_size*(args.dim+2)*4)*100*args.num_client/2
+    print("Local push Throughput (MB): %f" % (total_bytes / (end-start) / 1024.0 / 1024.0))
+    
+
+    ################################# remote push ######################################
+
+    
+    if args.machine_id == 0:
+        id_tensor = np.random.randint(args.graph_size, size=args.data_size)
+        id_tensor = id_tensor + args.graph_size
+    else:
+        id_tensor = np.random.randint(args.graph_size, size=args.data_size)
+    id_tensor = F.tensor(id_tensor)
+
+    kvclient.barrier()
+    start = time.time()
+    for _ in range(100):
+        res = kvclient.push(name='data', id_tensor=id_tensor, data_tensor=data_tensor)
+    kvclient.barrier()
+    end = time.time()
+    total_bytes = (args.data_size*(args.dim+2)*4)*100*args.num_client/2
+    print("Remote push Throughput (MB): %f" % (total_bytes / (end-start) / 1024.0 / 1024.0))
+    
+
+    dgl.distributed.shutdown_servers()
+    dgl.distributed.finalize_client()
+
+class ArgParser(argparse.ArgumentParser):
+    def __init__(self):
+        super(ArgParser, self).__init__()
+
+        self.add_argument('--range', type=int, default=1,
+                          help='If use range partition.')
+        self.add_argument('--num_client', type=int, default=2,
+                          help='Total number of clients.')
+        self.add_argument('--num_machine', type=int, default=2,
+                          help="number of machine.")
+        self.add_argument('--machine_id', type=int, default=0,
+                          help="machine ID.")
+        self.add_argument('--server_id', type=int, default=-1,
+                          help='server_id')
+        self.add_argument('--ip_config', type=str, default='ip_config.txt',
+                          help='IP configuration file of kvstore.')
+        self.add_argument('--data_size', type=int, default=100000,
+                          help='data_size of each machine.')
+        self.add_argument('--dim', type=int, default=10,
+                          help='dim of each data.')
+        self.add_argument('--graph_size', type=int, default=1000000,
```

Comment:
it looks small. can you benchmark on a larger graph? for example, 100 times larger?